script.js

```diff
@@ -0,0 +1,163 @@
+const buttons = document.querySelectorAll("button");
+const display = document.querySelector(".display");
+
+let firstNumber = "";
+let operator = "";
+let secondNumber = "";
+let result = "";
+
+const operate = (operator, a, b) => {
+  switch (operator) {
+    case "+":
+      return a + b;
+    case "-":
+      return a - b;
+    case "*":
+      return a * b;
+    case "/":
+      if (b === 0) {
+        return "Cannot divide by zero!";
+      } else {
+        return a / b;
+      }
+  }
+};
+
+const clearDisplay = () => {
+  display.textContent = "0";
+  firstNumber = "";
+  operator = "";
+  secondNumber = "";
+  result = "";
+};
+
+const updateDisplay = (value) => {
+  if (display.textContent === "0" || result !== "") {
+    display.textContent = value;
+    result = "";
+  } else {
+    display.textContent += value;
+  }
+};
+
+const backspace = () => {
+  switch (true) {
+    case secondNumber !== "":
+      secondNumber = secondNumber.slice(0, -1);
+      break;
+    case operator !== "":
+      operator = "";
+      display.textContent = firstNumber;
+      break;
+    case firstNumber !== "":
+      firstNumber = firstNumber.slice(0, -1);
+      break;
+    default:
+      display.textContent = "0";
+      break;
+  }
+
+  display.textContent =
+    firstNumber === "" && operator === "" && secondNumber === ""
+      ? "0"
+      : `${firstNumber}${operator}${secondNumber}`;
+};
+
+buttons.forEach((button) => {
+  button.addEventListener("click", (e) => {
+    const buttonValue = e.target.value;
+
+    switch (true) {
```

Comment:
You shouldn't use such an open switch.

Resort to if if there's no common expression